tests/kernel/arm_irq_vector_table/src/arm_irq_vector_table.c

```diff
@@ -114,25 +156,39 @@ void test_arm_irq_vector_table(void)
 
 }
 
-#if defined(CONFIG_SOC_SERIES_NRF52X)
+#if defined(CONFIG_SOC_SERIES_NRF52X) || defined(CONFIG_SOC_SERIES_NRF91X)
 /* nRF52X-based platforms employ a Hardware RTC peripheral
  * to implement the Kernel system timer, instead of the ARM Cortex-M
  * SysTick. Therefore, a pointer to the timer ISR needs to be added in
  * the custom vector table to handle the timer "tick" interrupts.
+ *
+ * The same applies to the CLOCK Control peripheral, which may trigger
+ * IRQs that would need to be serviced.
  */
 void rtc1_nrf_isr(void);
+void _power_clock_isr(void);
 typedef void (*vth)(void); /* Vector Table Handler */
+#if defined(CONFIG_SOC_SERIES_NRF52X)
 vth __irq_vector_table _irq_vector_table[RTC1_IRQn + 1] = {
+	_power_clock_isr,
```

Comment:
Maybe it would be good to rename this function to `nrf_power_clock_isr` to emphasize that it is nRF-specific?

Reply:
Sure. then we need to rename it in the driver file, too.

Reply:
Fixing this now.